src/main/resources/application.yml

```
@@ -0,0 +1,15 @@
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: ${JWT_AUTH_URL}
+server:
+  port: ${SERVER_PORT}
+paladin:
+  security:
+    jwt-secret-key: ${JWT_SECRET_KEY}
+    jwt-issuer: ${JWT_ISSUER}
+  include-stack-trace: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)